examples/imageviewer/mapview-desktop/src/desktopMain/kotlin/example/map/GeoPoint.kt

```diff
@@ -0,0 +1,48 @@
+package example.map
+
+import kotlin.math.*
+
+/**
+ * GeoPoint in relative geo coordinated
+ * x in range 0f to 1f (equals to longitude -180 .. 180)
+ * y in range 0 to 1  (equals to latitude 90 .. -90)
+ */
+data class GeoPoint(val x: Double, val y: Double)
+
+/**
+ * DisplayPoint screen coordinates (Also it may be used as distance between 2 screen point)
+ */
+data class DisplayPoint(val x: Int, val y: Int)
+
+val GeoPoint.longitude get():Double = x * 360.0 - 180.0
```

Comment:
Please apply auto-format. It looks like some spaces were missed

Reply:
Done